packages/perspective-viewer-d3fc/src/js/tooltip/generateHTML.js

```diff
@@ -32,7 +32,7 @@ function addDataValues(tooltipDiv, values) {
 }
 
 const formatNumber = (value) =>
-    value === null
+    value === null || value === undefined
```

Comment:
This is fine but just out of curiosity does this actually do anything?  Perspective's APIs _should not_ return `undefined`, only `null` for missing cells (in theory).